_config.yml

```diff
@@ -55,3 +58,13 @@ plugins:
 #   - vendor/cache/
 #   - vendor/gems/
 #   - vendor/ruby/
+
+#### jekyll-datapage-generator config
+
+page_gen:
+  - data: taxon_list
+    template: taxon
+    dir: taxa
+    name: genus
+    index_files: true
+#   debug: true
```

Comment:
As with the jekyll-datapage-generator plugin code in the Gemfile, this configuration should be located in theme's config file since generating pages for taxa should be considered core functionality of our theme.

Reply:
That's fine by me. My comfort level would be higher if you could do this addition, @alancleary - since this is on the theme side, which I haven't really worked on yet.